.github/workflows/test_suite_linux.yml

```diff
@@ -112,6 +112,7 @@ jobs:
       run: |
         export LIBGL_ALWAYS_SOFTWARE=true
         export WEBOTS_DISABLE_SAVE_SCREEN_PERSPECTIVE_ON_CLOSE=true
+        xvfb-run --auto-servernum python scripts/packaging/update_worlds_and_protos.py develop
```

Comment:
How long does it take to run this command?
If it takes long, then it should be moved in a different step (with dedicated label).
Something like it is done for the `world-update` step.

Reply:
It takes 2 seconds on my laptop, so I guess it's OK.

Reply:
When testing locally, for me it fails because of missing sha.
How do you make it work locally?

Reply:
You should pass `develop` instead of sha.

Reply:
Ok. I did once but not the second time. So it seems to work.
However here it seems wrong to pass develop instead of the sha.

Reply:
Yes, I believe we should pass the tag instead.

Reply:
Actually, this is probably the reason why the test was failing... I will re-enable it and see whether it works now.

Reply:
Running this script before the tests, are the tests still checking the internal functionality of Webots to convert the `webots://` URL?

Reply:
No, and this is probably not something we want to test here because:
- There is no image to test under webots:// included in the distribution that we use for testing.
- The worlds with a `webots://` URL are not distributed to users, but used only in the development environment, so it shouldn't affect users, but only developers and if it fails, we will probably spot it immediately.